android/databases/src/main/java/com/coinbase/wallet/libraries/databases/db/Storage.kt

```diff
@@ -242,6 +242,9 @@ internal class Storage<P : RoomDatabaseProvider> private constructor() {
             // Force a wal checkpoint which would transfer all transactions from the WAL file into the original DB.
             // This step is just a precaution to make sure no data is left in the original DB file.
             provider.query(SimpleSQLiteQuery("pragma wal_checkpoint(full)"))
+
+            // delete db files on file
+            disk.context.deleteDatabase(disk.dbName)
```

Comment:
@sosolove-dolg thanks for the contribution! We noticed that adding this line was causing some weird behavior where tables wouldn't get recreated after file deletion. Is there a reason you wanted to delete the file?

Reply:
1